docker/Dockerfile

```diff
@@ -0,0 +1,45 @@
+FROM python:3.5-alpine
+
+ARG ARTHUR_VERSION
+
+ENV \
+  ARTHUR_HOST="0.0.0.0"\
+  ARTHUR_PORT="8080" \
+  ARTHUR_DATABASE="redis://localhost/8" \
+  ARTHUR_LOGS_DIR="/arthur/logs"
+
+EXPOSE 8080
+
+LABEL \
+  maintainer="Chaoss" \
+  description="Arthur is a distributed job queue platform that schedules and executes Perceval" \
+  project="https://github.com/chaoss/grimoirelab-kingarthur" \
```

Comment:
Probably we should use King Arthur here. But not totally sure. In https://github.com/chaoss/grimoirelab-kingarthur in some places we are just using Arthur. @sduenas what is your opinion?